django/db/backends/base/schema.py

```diff
@@ -710,7 +710,7 @@ def _alter_field(self, model, old_field, new_field, old_type, new_type,
             )
         # Does it have a foreign key?
         if new_field.rel and \
```

Comment:
Since you are touching this if condition already, can you change the code formatting:

```python
if (new_field.rel and
        (fks_dropped ...) and
        new_field.db_constraint):
    # continue here
```